lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-address.yaml

```diff
@@ -45,6 +45,14 @@ states:
 
   # Journey States
 
+  RESET_IDENTITY_ADDRESS_CHANGE:
+    response:
+      type: process
+      lambda: reset-session-identity
+    events:
+      next:
+        targetState: ADDRESS_AND_FRAUD_UPDATE_ADDRESS
+
   ADDRESS_AND_FRAUD_UPDATE_ADDRESS:
     nestedJourney: ADDRESS_AND_FRAUD
     exitEvents:
```

Comment:
I can't comment on the actual line, but the `EVALUATE_GPG45_SCORES_UPDATE_ADDRESS` state needs to route to `STORE_IDENTITY_BEFORE_SUCCESS` and drop the `sessionCredentialsTableReads` flag.

The `CRI_TICF_BEFORE_SUCCESS_UPDATE_ADDRESS` state can also route directly to the `STORE_IDENTITY_BEFORE_SUCCESS` state without the flag.